installation/manual-installation/migrating-meteor-mongo/README.md

```diff
@@ -0,0 +1,19 @@
+# Migrating database from Meteor built in MongoDB
+
+This is a guide to migrate a manual installation database that is using the mongo database that comes built in meteor to an external mongo data base.
+
+We advise against using meteor's built in mongo on a production environment due performance reasons.
```

Comment:
I'd maybe start off at the top with something like:

> We advise against running Rocket.Chat in production from source using the meteor command.  Instead we recommend using a separate mongo database and using a compiled meteor bundle.  Either our official release or your own.

Idk something like that.  We should probably add something similar on any page that talks about using the meteor command :grin: